### ISSUE TITLE
Renaming the Live feature title within System --> General Settings

### ISSUE THREAD

**Issue by Chardonneaur** [piwik/piwik#21923]
Hi team,
Currently, the GDPR feature to disable/enable the visit log feature is named Live under System --> General Settings.
It makes it hard to identify what it does.
Would it be possible to rename it in something more self explanatory?

Thank you,

**Comment by Stan-vw** [comment]
Adding a screenshot of the current wording for easy reference:
<img width="1273" alt="Screen Shot 2024-03-11 at 7 09 26 PM" src="https://github.com/matomo-org/matomo/assets/52564377/5da7436a-1451-4310-bfa1-2a39dccea273">

I'd agree that the header isn't very descriptive. 

My suggestion would be "Real-time data settings", which seems in line with the rest of the headers' naming convention.
@Javi-Ormaechea or @ronak-innocraft Can you sense check?

**Comment by ronak-innocraft** [comment]
@Chardonneaur Is this documented on our user guides on where to find it and how to configure?

**Comment by Chardonneaur** [comment]
@ronak-innocraft yes, step 10 here https://matomo.org/faq/general/configure-privacy-settings-in-matomo/

**Comment by Javi-Ormaechea** [comment]
@ronak-innocraft & @Stan-vw that makes sense, maybe just "Real-time data" as it's found under Settings already?

**Comment by sgiehl** [comment]
Note: All settings displayed on the general settings page are settings for a plugin. The title displayed is currently always the plugin name. So to allow a custom title, we need to add a way for plugins to set a custom title for their system settings. In addition that name needs to be passed through when the settings are loaded through the API. So this will be a bit more effort than simply changing a text.

**Comment by Stan-vw** [comment]
If I recall correctly, changing the title of a plugin is time consuming as well, as it is used across various different places that then need to be adjusted as well.

**Comment by sgiehl** [comment]
Yes. That would be even more effort.

**Comment by mattab** [comment]
Lets not change it at this time
